presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergConfig.java

```diff
@@ -180,4 +181,17 @@ public boolean isMergeOnReadModeEnabled()
     {
         return mergeOnReadModeEnabled;
     }
+
+    public boolean isPushdownFilterEnabled()
```

Comment:
Shall we fail the query if this is enabled for the Java Iceberg execution? If the user enables it by accident, the query result would be incorrect for the Java worker.

Reply:
Yes, I had already added it in presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergPageSourceProvider.java